recipes/libsystemd/all/conanfile.py

```diff
@@ -50,13 +50,18 @@ def configure(self):
         del self.settings.compiler.cppstd
 
     def build_requirements(self):
-        if tools.Version(self.version) >= "248.3":
-            self.build_requires("meson/0.59.1")
+        if tools.Version(self.version) >= "249.5":
+            self.build_requires("meson/0.60.2")
+        elif tools.Version(self.version) >= "248.3":
```

Comment:
The breaking change is easy enough to patch as it seems to affect a single line in `meson.build`. What do you think? Do we patch the sources or keep the if-else chain?
```py
        tools.replace_in_file(
            meson_build,
            "if want_ossfuzz + want_libfuzzer > 1",
            "if want_ossfuzz and want_libfuzzer"
            )
```

Reply:
as it's well documented and only affects build tools, I see no problem patching it.